lib/dataTableHelpers.tsx

```diff
@@ -132,7 +136,12 @@ export function getAtlasColumn(atlases: Atlas[]) {
     const atlasMap = _.keyBy(atlases, (a) => a.htan_id);
 
     return {
-        name: 'Atlas Name',
+        id: 'Atlas Name',
+        name: (
+            <Tooltip overlay="Name of the Atlas">
```

Comment:
@inodb Can you think of a better description for Atlas Name?

Reply:
lol seems fine